inscriptions/ISic000294.xml

```diff
@@ -186,18 +191,22 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#TNS">
+                <ab>
+                <lb n="1"/><name><expan><abbr>L</abbr><ex>ucius</ex></expan></name> <g ref="#interpunct">·</g> <name><expan><abbr>Rosci</abbr><ex>us</ex></expan></name>
+                <lb n="2"/><name>Melanio</name>
+                <lb n="3"/>pius
+                <lb n="4"/>salve
```

Comment:
an interesting / tricky one.
Melanio implies that the name of the deceased is in the dative. So it should be L(ucio) Rosci(o) Melanio; but then pius is in the nominative, implying an epithet applied to him as a grammatically separate statement; and finally 'salve', in the same mode as the common Greek 'chaire', is almost certainly addressing the deceased, 'farewell', rather than the other way around (as evidenced by a number of this form where the name of the deceased is in the vocative - but while 'Rosci' might be that, Melanio can't be, and one would then get 'pie' and not 'pius'. But cases seem to get mixed around quite remarkably in funerary inscriptions, and frequently switch from indirect object to subject without obvious sentence structure to go with it. So, it hink dative, then nominative, and then an address...

Reply:
Thanks for this! Point noted about salve. I'll fix up my translation.

A case could be made for Melanio being a 3rd d. cognomen. A quick search on C/S reveals a bloke from Domavium called C. Iulius Silvanus Melanio (here in dat. Melanioni) (CIL 3.12732). A couple of other Iulii share the name. But Melanius is also attested at least once (AE 2001, 1389). Happy to defer to your judgement, though!